src/operations/Completion.cpp

```diff
@@ -1004,15 +781,15 @@ std::vector<lsp::CompletionItem> WorkspaceFolder::completion(const lsp::Completi
     {
         if (result.context == Luau::AutocompleteContext::Expression || result.context == Luau::AutocompleteContext::Statement)
         {
-            suggestImports(moduleName, position, config, *textDocument, items, /* includeServices: */ true);
+            suggestImports(moduleName, position, config, *textDocument, items, /* isType: */ false);
         }
         else if (result.context == Luau::AutocompleteContext::Type)
         {
             // Make sure we are in the context of completing a prefix in an AstTypeReference
             if (auto node = result.ancestry.back())
                 if (auto typeReference = node->as<Luau::AstTypeReference>())
                     if (!typeReference->prefix)
-                        suggestImports(moduleName, position, config, *textDocument, items, /* includeServices: */ false);
+                        suggestImports(moduleName, position, config, *textDocument, items, /* isType: */ true);
```

Comment:
nit: could we have a more descriptive name for `isType`? maybe `completingTypeReferencePrefix`, or something else